packages/orbit-components/src/Accordion/AccordionSection/components/SectionFooter.js

```diff
@@ -11,6 +11,7 @@ const Wrapper = styled.div`
   background-color: ${({ theme }) => theme.orbit.paletteWhite};
   position: sticky;
   bottom: 0;
+  box-sizing: border-box;
```

Comment:
How does this help? I tried turning it off and on and it doesn't seem to make a difference.